docs/networking.md

```diff
@@ -11,91 +11,121 @@ weight=4
 
 # Networking
 
-Docker Swarm is fully compatible for the new networking model added in docker 1.9
+Docker Swarm is fully compatible with Docker's networking features. This
+includes the mult-host networking feature which allows creation of a custom
```

Comment:
also "of a custom" -> "of custom" (we allow creation of multiple custom networks)